Mrs_Cake/Controllers/UsersController.cs

```diff
@@ -50,9 +48,9 @@ public ActionResult<User> Create(User user)
             return CreatedAtRoute("GetUser", new { id = user.Id.ToString() }, user);
         }
         [Route("login")]
-        [HttpGet]
+        [HttpPost]
```

Comment:
You use Post method for API but get method for user service and in code you don't send anything in DB. Post methods are user to push info to DB, GET methods = to pull info from DB